src/components/PopupWithForm.js

```diff
@@ -30,6 +30,14 @@ class PopupWithForm extends Popup {
     });
   }
 
+  renderLoading(isLoading) {
+    if (isLoading) {
+      this._popupForm.querySelector(".modal__button").textContent = "Saving...";
```

Comment:
Not all the popups have `Save` initial text. Please make the logic for rendering loading here as you did in the confirmation popup

Reply:
Please fix this issue